src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -158,6 +158,12 @@ var DOMPropertyOperations = {
         } else if (propertyInfo.hasBooleanValue ||
                    (propertyInfo.hasOverloadedBooleanValue && value === true)) {
           node.setAttribute(attributeName, '');
+        } else if (attributeName === 'value' && node.hasAttribute('value')) {
+          // Use loose coercion to prevent replacement on comparisons like
+          // '3e1' == 30 in Chrome (~52).
+          if (node.value != value) { // eslint-disable-line
+            node.setAttribute(attributeName, '' + value);
```

Comment:
Hmm. So actually: _should the value update for anything other than `checkbox` and `radio`?_

Following browser behavior, editing the text inside of a text input doesn't update the `value` attribute. 

Possible my current codes doesn't do this, but to me that's the correct functionality.